insights/parsers/mongod_conf.py

```diff
@@ -1,13 +1,15 @@
 """
-MongodbConf - files ``/etc/mongod.conf``, ``/etc/mongod.conf`` and ``/etc/opt/rh/rh-mongodb26/mongod.conf``
-===========================================================================================================
+MongodbConf - files ``/etc/mongod.conf``, ``/etc/mongod.conf``, ``/etc/opt/rh/rh-mongodb26/mongod.conf`` and ``/etc/opt/rh/rh-mongodb34/mongod.conf``
+=====================================================================================================================================================
```

Comment:
Better make the title a brief introduction of the parser, but not to reflect each of the files, the following doc will do that in detail.

e.g.
```
MongodbConf - files - Configuration files for MongDB
```

Reply:
Done, Please review.

Reply:
NIT: Sorry for the original typo, but it should be `MongoDB`

Reply:
Fixed . ;) I copied blindly